data/organisations.csv

```diff
@@ -0,0 +1,16 @@
+﻿organisation,abbrev

```

Comment:
The [schema](https://github.com/alphagov/api-catalogue/blob/master/data/organisation.csv-metadata.json) differs in column naming. Also the name of the file should be `organisation.csv` (i.e. in singular).

Reply:
Also, you need to add a synthetic identifier for every organisation as we can't rely on the name nor the acronym.